docs/guides/high-availability/introduction.adoc

```diff
@@ -12,6 +12,101 @@ Those setups are intended for a transparent network on a single site.
+== Tested Configuration
+
+We regularly test {project_name} with the following configuration:
+</@profile.ifCommunity>
+<@profile.ifProduct>
+== Supported Configuration
+</@profile.ifProduct>
+
+* Two Openshift single-AZ clusters, in the same AWS Region
+** Provisioned with https://www.redhat.com/en/technologies/cloud-computing/openshift/aws[Red Hat OpenShift Service on AWS] (ROSA),
+<@profile.ifProduct>
+either ROSA HCP or ROSA classic.
+</@profile.ifProduct>
+<@profile.ifCommunity>
+using ROSA HCP.
+</@profile.ifCommunity>
+
+** Each Openshift cluster has all its workers in a single Availability Zone.
+** OpenShift version
+<@profile.ifProduct>
+4.16 (or later).
+</@profile.ifProduct>
+<@profile.ifCommunity>
+4.16.
+</@profile.ifCommunity>
+
+* Amazon Aurora PostgreSQL database
+** High availability with a primary DB instance in one Availability Zone, and a synchronously replicated reader in the second Availability Zone
+** Version ${properties["aurora-postgresql.version"]}
+* AWS Global Accelerator, sending traffic to both ROSA clusters
+* AWS Lambda
+<@profile.ifCommunity>
+triggered by ROSA's Prometheus and Alert Manager
+</@profile.ifCommunity>
+to automate failover
+
+<@profile.ifProduct>
+Any deviation from the configuration above is not supported and any issue must be replicated in that environment for support.
+</@profile.ifProduct>
+<@profile.ifCommunity>
+While equivalent setups should work, you will need to verify the performance and failure behavior of your environment.
+We provide functional tests, failure tests and load tests in the https://github.com/keycloak/keycloak-benchmark[Keycloak Benchmark Project].
+</@profile.ifCommunity>
+
+Read more on each item in the <@links.ha id="bblocks-multi-site" /> {section}.
+
+<@profile.ifProduct>
+== Maximum load
+</@profile.ifProduct>
+<@profile.ifCommunity>
+== Tested load
+
+We regularly test {project_name} with the following load:
+</@profile.ifCommunity>
+
+* 100,000 users
+* 300 requests per second
+
+<@profile.ifCommunity>
+While we did not see a hard limit in our tests with these values, we ask you to test for higher volumes with horizontally and vertically scaled {project_name} name instances and databases.
+</@profile.ifCommunity>
+
+See the <@links.ha id="concepts-memory-and-cpu-sizing" /> {section} for more information.
+
+== Limitations
+
+<@profile.ifCommunity>
+Event with the additional redundancy of the two sites, downtimes can still occur:
+</@profile.ifCommunity>
+
+* During upgrades of {project_name} or {jdgserver_name} both sites needs to be taken offline for the duration of the upgrade.
+* During certain failure scenarios, there may be downtime of up to 5 minutes.
+* After certain failure scenarios, manual intervention may be required to restore redundancy by bringing the failed site back online.
+* During certain switchover scenarios, there may be downtime of up to 5 minutes.
+
+For more details on limitations see the <@links.ha id="concepts-multi-site" /> {section}.
```

Comment:
I believe this would be something we could talk about explicitly here.

```
Two sites restriction
This setup is tested and supported only with two sites. Each additional site increases overall latency as it is necessary for data to be synchronously written to each site. Furthermore, the probability of network failures, and therefore downtime, also increases. Therefore, we do not support more than two sites as we believe it would lead to a deployment with inferior stability and performance.

```

Reply:
This is taken from the previously agreed text and will therefore not be changed.